src/main/java/pivotal/ui/pages/ProfilePage.java

```diff
@@ -0,0 +1,149 @@
+package pivotal.ui.pages;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.function.Supplier;
+
+import core.utils.WebElementInteractor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pivotal.entities.User;
+
+import static pivotal.constants.UserIdentifiers.NAME;
+import static pivotal.constants.UserIdentifiers.USER_NAME;
+import static pivotal.constants.UserIdentifiers.INITIALS;
+
+public class ProfilePage extends BasePage {
+    @FindBy(id = "person_username")
+    private WebElement usernameTextBox;
+
+    @FindBy(id = "person_name")
+    private WebElement nameTextBox;
+
+    @FindBy(id = "person_initials")
+    private WebElement initialsTextBox;
+
+    @FindBy(xpath = "//div[contains(text(),'UpdatedUsername')]")
+    private WebElement usernameField;
+
+    @FindBy(xpath = "//ul[@class='rows read']//li//div[contains(text(),'Updated name')]")
+    private WebElement nameField;
+
+    @FindBy(css = "div.profile_page.settings:nth-child(3) div.card:nth-child(1) ul.rows.read li:nth-child(3) > div:nth-child(2)")
+    private WebElement initialsField;
+
+    @FindBy(css = "#person_start_page")
+    private WebElement startPageDropDown;
+
+    @FindBy(css = "#person_time_zone")
+    private WebElement timeZoneDropDown;
+
+    @FindBy(css = "#person_default_story_type_value")
+    private WebElement defaultStoryTypeDropDown;
+
+    @FindBy(css = "#general .save_button")
+    private WebElement saveBtn;
+
+    @FindBy(css = "#general .edit_button")
+    private WebElement editBtn;
+
+    @FindBy(css = "#general_flash span")
+    private WebElement changesNotification;
+
+    @FindBy(css = ".name")
+    private WebElement userMMTitle;
+
+    /**
+     * Instantiates a new Profile page.
+     */
+    public ProfilePage() {
+        super();
+    }
+
+    private void setInformationToEdit(final Set<String> formFields, final User user) {
+        HashMap<String, Runnable> strategyMap = composeMyProfileMap(formFields, user);
+        formFields.forEach(key -> strategyMap.get(key).run());
+    }
+
+    /**
+     * Edit profile.
+     *
+     * @param username the username
+     * @param name     the name
+     * @param initials the initials
+     */
+    public void editProfile(final String username, final String name, final String initials) {
```

Comment:
Where do you use this method? if you have the other one editProfile, you should choose one or add a comment to explain when this mehtod should be used and when it is better to use the other one